src/Apps/gEvGen.cxx

```diff
@@ -789,33 +753,10 @@ void GetCommandLineArgs(int argc, char ** argv)
          gOptTgtMix.insert(map<int, double>::value_type(pdg, wgt));
       }//tgtmix_iter
     }//>1
-
```

Comment:
Probably want to keep this section with the error note that one _must_ specify the target PDG

Reply:
I made the neutrino PDG and the target argument required with [84f1030](https://github.com/GENIE-MC/Generator/pull/322/commits/84f1030dc38ca03d2539340dac6532d6b655911a)